recipes/wasmtime/all/test_package/example.cpp

```diff
@@ -0,0 +1,10 @@
+#include <thread>
```

Comment:
Since this is a C api , you should change the file extension and make sure it does not require the STL

Reply:
Done